.github/workflows/ci.yml

```diff
@@ -642,6 +642,40 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  benchmarks:
+    name: Benchmarks
+    runs-on: ubuntu-latest
+    needs: check
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+
+      - name: Download previous benchmark data
+        uses: Swatinem/rust-cache@v2
+        with:
+          key: ${{ runner.os }}-benchmark
+
+      - name: Run Benchmarks
+        run: cargo +nightly bench | tee benchmarks.txt
+
+      - name: Store benchmark result
+        uses: benchmark-action/github-action-benchmark@v1
```

Comment:
I think that we need the parameter `external-data-json-path`

Reply:
Yeah, I just added support to cache a `benches` directory where we will store and gather previous benchmark.